util/upload-instance-slot-map.py

```diff
@@ -99,6 +152,12 @@ def upload(regions, main_region, credentials):
                     pass
 
 
+def __fail(msg):
```

Comment:
minor: 
We are using the convention to have a single underscore as prefix of module's private functions.
We use double underscore for private class methods.